Slim/Http/Response.php

```diff
@@ -435,7 +435,7 @@ public function isClientError()
      * Helpers: Server Error?
      * @return bool
      */
-    public function isServerError()
+    public function isServerError(): bool
```

Comment:
```suggestion
    public function isServerError()
```

This one is not necessary for PHP 8.1 compatibility (so I'd rather have it remain as it is).